### PR TITLE
Preparing 1.2 release

### PR DESCRIPTION
**Pull request by alanz**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -20,89 +65,89 @@ Once that hurdle is crossed, the main emphasis will be there, and
 ## Changes
 
 - Install script with cabal: check there is one ghc in $PATH
-([#1632](https://github.com/haskell/haskell-ide-engine/pull/1551), by @jneira)
+([#1632](https://github.com/haskell/haskell-ide-engine/pull/1632), by @jneira)
```
Comment (fendor) [inline Changelog.md]: Oh my, did we all miss that?

Comment (alanz) [reply Changelog.md]: Yes. I was horrified when I saw all the link destinations were the same.  It was a rush job :(

Comment (fendor) [reply Changelog.md]: Thank you for fixing it! so many PRs :O

File: Changelog.md [hunk 2]
```diff
@@ -1,3 +1,48 @@
...
+Once that hurdle is crossed, the main emphasis will be there, and
+`hie` will eventually be deprecated.
+
+## In this version
+
+- cabal now index state 2020-03-01T07:49:28Z
+- GHC 8.8.2 is lts-15.2
+- GHC 8.6.5 is lts-14.22
+- hlint is 2.2.11
+- brittany is 0.12.1.1
```
Comment (fendor) [inline Changelog.md]: Maybe I am missing something, but isnt in this PR the lower haskell-src-exts version bumped as well?

**Pull request merged**